qdrant-landing/themes/qdrant-2024/layouts/qdrant-pricing/baseof.html

```diff
@@ -30,5 +30,8 @@
   {{ partial "get-contacted" (dict "context" . "class" "qdrant-pricing-get-contacted") }}
 {{ end }}
 
-{{ partial "qdrant-pricing-docker" . }}
+{{ with (.Site.GetPage "/headless/main/docker-deploy") }}
+  {{ partial "docker-deploy" (dict "context" . "class" " qdrant-pricing-docker-deploy")}}
```

Comment:
Why do we need this class different? I could not spot any visual differences in those two components

Reply:
text align for title and subtitle in mobile version